appveyor.yml

```diff
@@ -15,32 +12,28 @@ install:
   - git submodule update --init --recursive
 
 environment:
+  app_version: "2.0"
   USE_RTOOLS: true
   NOT_CRAN: true
   KEEP_VIGNETTES: true
   R_ARCH: x64
   matrix:
-  - R_VERSION: devel
-    GCC_PATH: mingw_64
+  # - R_VERSION: devel
+  #   GCC_PATH: mingw_64
   - R_VERSION: release
   - R_VERSION: oldrel
 
+version: "$(app_version).{build}"
+
 before_build:
-  - nuget sources add -name utility -source https://ci.appveyor.com/nuget/ospsuite-utility
-  - nuget sources add -name serializer -source https://ci.appveyor.com/nuget/ospsuite-serializer
-  - nuget sources add -name funcparser -source https://ci.appveyor.com/nuget/ospsuite-funcparser
-  - nuget sources add -name simmodel -source https://ci.appveyor.com/nuget/ospsuite-simmodel
-  - nuget sources add -name cvodes -source https://ci.appveyor.com/nuget/ospsuite-simmodel-solver-cvodes
-  - nuget sources add -name core -source https://ci.appveyor.com/nuget/ospsuite-core
-  - nuget sources add -name centos -source https://ci.appveyor.com/nuget/ospsuite-centos
-  - nuget install packages.config -OutputDirectory packages # cannot use automatic artifact publish because it's not recognized in a C++/CLI project
   - rake "prepare_for_build[%APPVEYOR_BUILD_VERSION%]"
 
 build_script:
   - Rscript -e "install.packages('./tools/rClr_0.9.1.zip', repos = NULL, type = 'binary')"
```

Comment:
Can we reference the online version?:

`Rscript -e "install.packages('.https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1/rClr_0.9.1.zip', repos = NULL, type = 'binary')"`

It works locally in RStudio. This way we do not have to pack rClr in the package.